index.html

```diff
@@ -15,157 +15,200 @@
-                    <li><a href="#"><img src="./assets/icons/csslogo.png" alt="csslogo"></a></li>
-                    <li><a href="#"><img src="./assets/icons/jslogo.png" alt="jslogo"></a></li>
-                </ul>
-                <div class="button-container">
-                    <button class="button-see-project" type="button">
-                        See project
-                    </button>
-                </div>
             </article>
 
             <!-- Second Page -->
 
             <article class="work-section">
 
-                <img src="./assets/images/Multi-post-stories.png" alt="Snapshot Portfolio">
-
-
-                <h1 class="project-title-heading">Multi-Post <br> Stories</h1>
-
-                <article class="frame-2">
-                    <h2 class="client-heading">CANOPY</h2>
-                    <img src="./assets/icons/Counter.png" alt="Counter">
-                    <h2 class="role">Back End Dev</h2>
-                    <img src="./assets/icons/Counter.png" alt="Counter">
-                    <h2 class="year">2015</h2>
+                <img src="./assets/images/Multi-post-stories.png" alt="Snapshot Portfolio"
+                    class="portfolio-img-1-mobile">
+                <article class="work-section-texts">
+                    <h1 class="project-title-heading">Multi-Post <br> Stories</h1>
+
+                    <article class="frame-2">
+                        <h2 class="client-heading text-mobile">CANOPY</h2>
+                        <h2 class="client-heading text-desktop">FACEBOOK</h2>
+                        <img src="./assets/icons/Counter.png" alt="Counter">
+                        <h2 class="role text-mobile">Back End Dev</h2>
+                        <h2 class="role text-desktop">Full Stack Dev</h2>
+                        <img src="./assets/icons/Counter.png" alt="Counter">
+                        <h2 class="year">2015</h2>
+                    </article>
+
+                    <p class="project-info-p text-mobile">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+
+                    <p class="project-info-p text-desktop">
+                        Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+                    </p>
+
+                    <ul class="logo-tags">
+                        <li><a href="#"><img src="./assets/icons/Htmllogo.png" alt="Htmllogo"></a></li>
+                        <li><a href="#"><img src="./assets/images/ruby-n-rails-tag.png" alt="Ruby on rails" class="portfolio-img-1-desktop"></a></li>
+                        <li><a href="#"><img src="./assets/icons/csslogo.png" alt="csslogo"></a></li>
+                        <li><a href="#"><img src="./assets/icons/jslogo.png" alt="jslogo"></a></li>
+                    </ul>
```

Comment:
Please, kindly align this element to stick to Figma.

Your design:
<img width="350" alt="image" src="https://user-images.githubusercontent.com/7243132/171745884-ca1acdd5-cc86-4175-93d6-1fa75e4fbc0a.png">

Figma:
<img width="367" alt="image" src="https://user-images.githubusercontent.com/7243132/171746052-b2a94cf6-8956-487f-baac-3842030a96ed.png">